packages/analytics/analytics-chart/src/components/chart-types/StackedBarChart.vue

```diff
@@ -139,24 +132,31 @@ const props = defineProps({
   },
 })
 
-const emit = defineEmits(['heightUpdate'])
-
 const { i18n } = composables.useI18n()
 const { translateUnit } = composables.useTranslatedUnits()
 
 // https://www.chartjs.org/chartjs-plugin-annotation/latest/guide/types/label.html#label-annotation-specific-options
 const LABEL_PADDING = 6
 
 // Parameters for bar sizing.
-const MIN_BAR_HEIGHT = 20
-const MIN_CHART_WIDTH = '100%'
+const DEFAULT_CHART_WIDTH = '100%'
+const DEFAULT_CHART_HEIGHT = '100%'
 const MAX_BARS_VERTICAL = 30
+const MAX_BARS_HORIZONTAL = 35
 const MIN_BAR_WIDTH = 40
 const SCROLL_MIN = 0
 const SCROLL_MAX = 10
 const AXIS_BOTTOM_OFFSET = 10
 const AXIS_RIGHT_PADDING = 1
-const BAR_MARGIN = 6
+
+const chartContainerRef = ref<HTMLDivElement>()
```

Comment:
issue: template ref needs to be initialized as null 
```suggestion
const chartContainerRef = ref<HTMLDivElement | null>(null)
```